modules/andes-core/broker/src/main/java/org/wso2/andes/configuration/enums/AndesConfiguration.java

```diff
@@ -560,7 +560,12 @@
      */
     PERFORMANCE_TUNING_MESSAGE_EXPIRATION_BATCH_SIZE
             ("performanceTuning/messageExpiration/messageBatchSize", "1000", Integer.class),
-
+    /**
```

Comment:
1 is not a safe margin. we will do a performance /long running tests and decide what a sensible value would be.